app/scripts/modules/core/pipeline/config/stages/jenkins/jenkinsStage.html

```diff
@@ -79,11 +79,31 @@ <h4 class="text-left" >Job Parameters</h4>
 
   </div>
 
-  <stage-config-field label="Fail Pipeline" help-key="pipeline.config.failPipeline">
-    <div class="checkbox">
-      <label>
-        <input type="checkbox" ng-model="stage.failPipeline">
+  <div class="form-group">
+    <label class="col-md-2 col-md-offset-1 sm-label-right">If job fails</label>
+    <div class="col-md-3" ng-init="viewState.failureOption=jenkinsStageCtrl.failureOptionInit()">
+      <label class="radio-inline col-md-offset-1">
+        <input type="radio"
+               name="failureOption"
+               ng-model="viewState.failureOption"
+               ng-change="jenkinsStageCtrl.failureOptionChanged()"
+               value="fail"/> Halt the entire pipeline
+      </label>
+      <label class="radio-inline col-md-offset-1">
+        <input type="radio"
+               name="failureOption"
+               ng-model="viewState.failureOption"
+               ng-change="jenkinsStageCtrl.failureOptionChanged()"
+               value="stop"/> Halt this branch of the pipeline
+      </label>
+      <label class="radio-inline col-md-offset-1">
```

Comment:
remove the `col-md-offset-1` class from the labels - that will fix the positioning